app/helpers/permissions_helper.rb

```diff
@@ -83,7 +83,7 @@ def set_user_gon
     gon.global.admin = current_user.admin
 
     # maybe find a more logical place for this
-    gon.global.api_url = ENV['API_URL']
+    gon.global.api_url = Rails.env.development? ? 'http://localhost:3000' : ENV['API_URL']
```

Comment:
I think this is an error. The `API_ENV` environment variables points to the RW API.